i18n/en.pot

```diff
@@ -50,6 +50,29 @@ msgstr ""
 msgid "selected"
 msgstr ""
 
+msgid "All {{count}} items on this page are selected."
+msgid_plural "All {{count}} items on this page are selected."
+msgstr[0] ""
+msgstr[1] ""
```

Comment:
So this is how plurals work, good to know! 

To sum it up: When a `i18n.t(msg, {count: N})` is found, gettext creates a msgid and msgid_plural, and -in the general case, see `Plural-Forms`-, `msgstr[0]` is the singular text and `msgstr[1]` the plural text.

https://www.gnu.org/software/gettext/manual/html_node/Translating-plural-forms.html